.github/workflows/build-test.yml

```diff
@@ -43,6 +43,7 @@ on:
 # When updating the CI stable version, please audit all cases of ICU4X_BUILDING_WITH_FORCED_NIGHTLY and remove cfgs where necessary.
 env:
   INSTALLED_NIGHTLY_VERSION: nightly-2022-04-05
+  RUST_CACHED_PATH_ROOT: /tmp/icu4x
```

Comment:
Question: Where is this being used?

Reply:
It's the directory where datagen downloads the zip files to.

Reply:
Used by the `cached_path` crate.

Reply:
That's weird. I thought the downloads were going into [`dirs::cache_dir()`](https://docs.rs/dirs/latest/dirs/fn.cache_dir.html); did this change? `/tmp` is not a great place to put them because the OS doesn't really make guarantees about how the lifecycle of tmp files. The OS could delete them while the process is still running even.

Reply:
Suggestion: add a `--download-dir` or `--cache-dir` command line argument if you don't already have one

Reply:
(Since this is pre-existing, it can be in another PR)

Reply:
> That's weird. I thought the downloads were going into [dirs::cache_dir()](https://docs.rs/dirs/latest/dirs/fn.cache_dir.html); did this change?

Yes, when we moved to `cached_path`. It behaves like [this](https://docs.rs/cached-path/latest/cached_path/struct.CacheBuilder.html#method.dir).

> The OS could delete them while the process is still running even.

Well, not while we have the file open

Reply:
I see; so the environment variable does actually have an effect here, `RUST_CACHED_PATH_ROOT`

That's better than nothing.

Reply:
That's why I'm setting it 😉